### DISCUSSION TITLE
Random Function

### DISCUSSION THREAD

**Discussion by mmediaman** [WarmUpTill/SceneSwitcher#818] [General]
Hello everyone. 
Sorry if this sounds like a rookie question, but I'm looking for a simple way to set up random scene switching that will resume its random function after being interrupted by a manual scene switch.  I used an older version of Adv Scene Switcher which did this by default.   Doesn't seem to work quite the same in versions of Adv Scene Switcher since Macros were introduced.

Any help appreciated.

**Comment by WarmUpTill** [comment]
As mentioned in the forums it depends a lot on what you want to exactly achieve.
Here is an example to get you started:
https://github.com/WarmUpTill/SceneSwitcher/wiki/Switch-scenes-randomly

Let me know if you have any questions!

(Note that the old tabs are also still available, but I would recommend to switch to macros instead)

**Reply by mmediaman** [reply]
Thank you much! Works perfectly!
My biggest mistake was not being open to using the Macros.
